tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ProcessStart/ProcessStartTags.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="ProcessStartTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.SourceGenerators;
+
+namespace Datadog.Trace.Tagging
+{
+    internal partial class ProcessCommandStartTags : InstrumentationTags
+    {
+        [Tag(Trace.Tags.SpanKind)]
+        public override string SpanKind => SpanKinds.Consumer;
+
+        [Tag(Trace.Tags.ProcessEnvironmentVars)]
+        public string EnviromentVars { get; set; }
+
+        [Tag(Trace.Tags.ProcessTruncated)]
+        public string Truncated { get; set; }
```

Comment:
Shouldn't this be a `bool`? 🤔 I'd probably call it `IsTruncated` anyway

Reply:
I have changed the name. About the type, I think that tags properties have to be a string.

Reply:
Oh yeah, you're right 🤦‍♂️ We've used "metrics" for booleans in the past instead